target-config.js

```diff
@@ -119,7 +119,8 @@
       webAnimations1BonusSrc: webAnimations1BonusSrc,
       webAnimationsNextSrc: [],
       src: scopeSrc.concat(sharedSrc).concat(webAnimations1Src).concat(webAnimations1BonusSrc),
-      test: webAnimations1Test,
+      polyfillTests: webAnimations1PolyfillTests,
+      runWebPlatformTests: false,
```

Comment:
On second thought I'd prefer to leave this to a separate patch to deal with the issue of expectation inheritance.